bulk_deploy/tsuru_api_service.rb

```diff
@@ -0,0 +1,119 @@
+require_relative './../dsl/ssh_helper'
+
+class TsuruAPIService
+
+  def initialize(logger:, api_client:, tsuru_home:)
```

Comment:
Aha! Yes - upgrade and that fixed the issue. I'll add a `.ruby-version` to the repo to be explicit about the ruby version in a separate PR.